src/aap_eda/services/project/imports.py

```diff
@@ -163,13 +163,17 @@ def _sync_rulebook(
         rulebook_info: RulebookInfo,
     ):
         if rulebook.rulesets == rulebook_info.raw_content:
+            models.Activation.objects.filter(rulebook=rulebook).update(
```

Comment:
Why do we need this line given it has been set by line 176? If it is for migration purpose, we should do it in the migration file.

Reply:
This is to make sure that we update the git hash for rulebook activations, even if the rulebook has not been changed but the project overall has changed.

Reply:
OK. You still need to enhance the migration file to set githash for existing activations.